ros-examples/src/aero_offboard_velocity/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package>
+  <name>aero_offboard_velocity</name>
+  <version>0.0.0</version>
+  <description>Demonstration of offboard velocity for Ned and body frames on Intel Aero</description>
```

Comment:
You can add description as *Example that demonstrates offboard velocity control in local NED and body coordinates* using MAVROS on Intel Aero